src/app/qgsmergeattributesdialog.cpp

```diff
@@ -319,12 +320,12 @@ void QgsMergeAttributesDialog::refreshMergedValue( int col )
     return;
   }
 
-  int fieldIdx = mTableWidget->horizontalHeaderItem( col )->data( FieldIndex ).toInt();
+  const int fieldIdx = mTableWidget->horizontalHeaderItem( col )->data( FieldIndex ).toInt();
 
   //evaluate behavior (feature value or min / max / mean )
-  QString mergeBehaviorString = comboBox->currentData().toString();
+  const QString mergeBehaviorString = comboBox->currentData().toString();
   QVariant mergeResult; // result to show in the merge result field
-  if ( mergeBehaviorString == QLatin1String( "concat" ) )
+  if ( mergeBehaviorString == QStringLiteral( "concat" ) )
```

Comment:
I think that's better to leave as before, but don't loose too much time on these micro-optimizations :)

https://doc.qt.io/qt-5/qstring.html#operator-eq-eq

Reply:
thanks, I missed that one